leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionServiceBuilder.java

```diff
@@ -84,6 +85,11 @@ public PaxosLeaderElectionServiceBuilder leaderPingResponseWaitMs(long leaderPin
         return this;
     }
 
+    public PaxosLeaderElectionServiceBuilder noQuorumMaxDelayMs(long noQuorumMaxDelayMs) {
+        this.noQuorumMaxDelayMs = noQuorumMaxDelayMs;
+        return this;
+    }
+
     public PaxosLeaderElectionServiceBuilder eventRecorder(PaxosLeaderElectionEventRecorder eventRecorder) {
         this.eventRecorder = eventRecorder;
         return this;
```

Comment:
I can't comment on the below `build()`, but do you need to pipe the `noQuorumMaxDelayMs` into the `PaxosLeaderElectionService` constructor?

Reply:
I agree that I do. Fixed, thanks.